x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -35,106 +37,127 @@ export const EnginesList: React.FC = () => {
   }, [meta.from, meta.size, searchQuery]);
 
   return (
-    <EnterpriseSearchContentPageTemplate
-      pageChrome={[
-        i18n.translate('xpack.enterpriseSearch.content.engines.breadcrumb', {
-          defaultMessage: 'Engines',
-        }),
-      ]}
-      pageHeader={{
-        pageTitle: i18n.translate('xpack.enterpriseSearch.content.engines.title', {
-          defaultMessage: 'Engines',
-        }),
-        rightSideItems: [
-          <EuiButton
-            fill
-            iconType="plusInCircle"
-            data-test-subj="enterpriseSearchContentEnginesCreationButton"
-            href={'TODO'}
-          >
-            {i18n.translate('xpack.enterpriseSearch.content.engines.createEngineButtonLabel', {
-              defaultMessage: 'Create engine',
-            })}
-          </EuiButton>,
-        ],
-      }}
-      pageViewTelemetry="Engines"
-      isLoading={false}
-    >
-      <EuiText>
-        {i18n.translate('xpack.enterpriseSearch.content.engines.description', {
-          defaultMessage:
-            'Engines allow you to query indexed data with a complete set of relevance, analytics and personalization tools. To learn more about how engines work in Enterprise search ',
-        })}
-
-        <EuiLink data-test-subj="documentationLink" href="TODO" target="_blank">
-          {i18n.translate('xpack.enterpriseSearch.content.engines.documentation', {
-            defaultMessage: 'explore our Engines documentation',
-          })}
-        </EuiLink>
-      </EuiText>
-      <EuiSpacer />
-      <div>
-        <EuiFieldSearch
-          value={searchQuery}
-          placeholder={i18n.translate('xpack.enterpriseSearch.content.engines.searchPlaceholder', {
-            defaultMessage: 'Search engines',
-          })}
-          aria-label={i18n.translate('xpack.enterpriseSearch.content.engines.searchBar.ariaLabel', {
-            defaultMessage: 'Search engines',
+    <>
+      <DeleteEngineModal />
+      <EnterpriseSearchContentPageTemplate
+        pageChrome={[
+          i18n.translate('xpack.enterpriseSearch.content.engines.breadcrumb', {
+            defaultMessage: 'Engines',
+          }),
+        ]}
+        pageHeader={{
+          pageTitle: i18n.translate('xpack.enterpriseSearch.content.engines.title', {
+            defaultMessage: 'Engines',
+          }),
+          rightSideItems: [
+            <EuiButton
+              fill
+              iconType="plusInCircle"
+              data-test-subj="enterprise-search-content-engines-creation-button"
+              data-telemetry-id={`entSearchContent-engines-list-createEngine `}
```

Comment:
I was not sure what should add in place of `ingestion type` for `data-telemetry-id`

Reply:
This is fine. Though you instead of using `{\`string content\`}` you can just do `"string-content"`